livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/realtime/converstation_items.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from collections import OrderedDict
+from dataclasses import dataclass, field
+from typing import Optional
+
+from livekit.agents import llm
+
+from .log import logger
+
+
+@dataclass
+class ConversationItem:
+    """A node in the conversation linked list"""
+
+    message: llm.ChatMessage
+    _prev: Optional[ConversationItem] = field(default=None, repr=False)
+    _next: Optional[ConversationItem] = field(default=None, repr=False)
+
+
+class ConversationItems:
```

Comment:
This is a double linked list to mimic the item operation in the Realtime API. Let me know if this is necessary or if there is any other tools for easy deleting and inserting items to the list. And this may need a rename to avoid confusion with the Conversation Items in the API.

Reply:
I think this is OK since this is internal. Maybe `RemoteConversationItems`?
Let're prepend '`_`' to the file name and classes to explicitly mark them as internal

Reply:
Sound good.